Makefile

```diff
@@ -65,12 +65,16 @@ all: build images
 
 run-docker: # @HELP run onos-config docker image
 run-docker: images
-	docker run -d -p 5150:5150 -v `pwd`/configs:/etc/onos-config \
+	docker stop onos-config
```

Comment:
this probably out to be part of clean, or his own step, including stop of cli, thoughts ?

Reply:
The reason I added it is that it is not possible run run it twice in a row without deleting container.

I'd like to leave it here **and** add it to **clean** without having to force **clean** to be run.

Reply:
changed to
```
docker stop onos-config || echo "onos-config was not running"
```